src/System.Private.CoreLib/shared/System/Runtime/Serialization/DeserializationBlockedException.cs

```diff
@@ -0,0 +1,42 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace System.Runtime.Serialization
+{
+    // Thrown when a dangerous action would be performed during deserialization 
+    [Serializable]
+    [System.Runtime.CompilerServices.TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
```

Comment:
I have not seen this change going into mscorlib / desktop. Do we really need this forward?

In any case, there should be a test to verify that the exception is binary serializable. I do not think it is the case with the current state of the changes.

Reply:
It will go into desktop at some point (likely after 4.8). I'll add a test for serializing it.